fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -593,7 +593,7 @@ protected CompletableFuture<Boolean> throttleDelayAndMaybeLogProgress(SubspacePr
                     LogMessageKeys.LIMIT, limit,
                     LogMessageKeys.DELAY, toWait,
                     LogMessageKeys.RECORDS_PER_SECOND, recordsPerSecond)
-                    .addKeysAndValues(additionalLogMessageKeyValues)
+                    .addKeysAndValues(additionalLogMessageKeyValues != null ? additionalLogMessageKeyValues : Collections.emptyList())
```

Comment:
This should probably have a new issue, and an update to the release notes.

Reply:
✅